scripts/downloadAndExtractEdge.js

```diff
@@ -69,16 +69,14 @@ async function downloadZipFile(downloadUrl) {
       fs.unlink('edge.zip', () => {} );
       let dirName = __dirname;
       if (isWindows) {
-        flipSlashDirName = dirName.replace(/\//g, '\\');
+        const flipSlashDirName = dirName.replace(/\//g, '\\');

```

Comment:
same here

Reply:
`downloadAndExtractEdge.js` is a standalone isolated script, so we cannot import the path library unless we list this as a module.  Don't know if we want to go down that route especially if we are already version checking for which Edge source files to download.

Reply:
you are right we are already checking for all the platforms :)